paper/paper.md

```diff
@@ -1,124 +1,237 @@
+
+Second, in addition to the new function-level model implementations,
+the package's high-level classes have also been expanded to support
+the complexity of emerging system designs, including heterogeneous systems whose
+subsystems differ in mounting or electrical configuration and systems that require
+custom orientation/tracking models.
+
+Third, the creation of `pvlib.iotools`, a sub-package for fetching and importing
+datasets relevant to PV modeling.  These functions provide a standardized
+interface for reading data files in various complex data formats, offering
+conveniences like optionally standardizing the dataset variable names and units
+to pvlib's conventions [@pvlibiotools].  As of version 0.10.1, `pvlib.iotools` contains
+functions to download data from over ten online data providers,
+plus file reading/parsing functions for a dozen solar resource file formats.
+
+These additions are discussed in more detail in [@pvpmc_2023_update] and [@pvpmc2022_pvlib_update].
+Complete descriptions of the changes in each release can be found in the
+project's documentation.
+
+
+# Community growth
+
+It is difficult to comprehensively describe the community around
+open-source projects like pvlib python, but some aspects can be
+quantified.  Here we examine the community from a few convenient
+perspectives, emphasizing that these metrics provide a limited view of
+the community as a whole.
+
+First, we examine contributors to pvlib python's code repository.  The
+project's use of version control software enables easy quantification of
+repository additions (to code, documentation, tests, etc) over time.  The
+project's repository currently comprises contributions from over 100 people
+spanning industry, academia, and government research institutions.
+\autoref{fig:community} (left) shows the number of unique repository
+contributors over time, demonstrating continued and generally accelerating
+attraction of new contributors.
+
+![Total repository contributor count over time (left) and other community size statistics (right).\label{fig:community}](community.pdf)
+
+However, the project as a whole is the product of not only of those who contribute
+code but also those who submit bug reports, propose ideas for new features,
+participate in online forums, and support the project in other ways.
+Along those lines, two easily tracked metrics are the number of people
+registered in the pvlib python online discussion forum and the number of
+GitHub "stars" (an indicator of an individual's interest, akin to a browser bookmark)
+on the pvlib python code repository.  \autoref{fig:community} (right)
+shows these counts over time.  Although these numbers
+almost certainly underestimate the true size of the pvlib community,
+their increase over time indicates continued and accelerating community growth.
```

Comment:
```suggestion
their increase over time indicates continued and accelerating community growth.
Since the 2018 JOSS publication, pvlib python has doubled the number of 
maintainers to bring in new perspectives and to better support the growing 
community.
```